docs/pieces.md

```diff
@@ -327,13 +322,13 @@ These are the important sub-sections of a query builder:
 
 ## REST APIs for pieces
 
-REST APIs are automatically made available for piece types. For more information, see [REST APIs](rest-apis.md).
+REST APIs are automatically made available for piece-types. For more information, see [REST APIs](rest-apis.md).
```

Comment:
I don't think this should be hyphenated. `piece-type` is a module name. "Piece type" is a concept.

Reply:
Correct, no hyphen.